mods/pvp/antisabotage/init.lua

```diff
@@ -12,10 +11,17 @@ minetest.register_on_dignode(function(pos, oldnode, digger)
 
 			if math.floor(player_pos.y) == pos.y and vector.distance(player_pos, pos) <= 1.5 then
 				minetest.set_node(pos, oldnode)
-				digger:get_inventory():remove_item("main", ItemStack(oldnode))
+				for _, item in pairs(minetest.get_node_drops(oldnode)) do -- Properly remove items dropped by nodes, now also protects against possible crash by nodes dropping more than one item.
```

Comment:
Make it shorter so it can be more readable for users.
```suggestion
				for _, item in pairs(minetest.get_node_drops(oldnode)) do -- Properly remove items dropped by nodes,
				-- now also protects against possible crash by nodes dropping more than one item.
```

Reply:
no